src/Common/Misc/ConvertProxyProperty/index.js

```diff
@@ -9,7 +9,7 @@ const typeMapping = {
   ProxyEditorPropertyWidget: 'ProxyEditorPropertyWidget',
 };
 
-function isGroupWidget(widgetType) {
+export function isGroupWidget(widgetType) {
```

Comment:
Well the other functions in the default export are also exported.  Doesn't it just give more freedom in terms of how it can be imported?